client/src/App.tsx

```diff
@@ -1,13 +1,23 @@
-import Home from "./pages/Home";
-import {Route, Routes} from "react-router-dom";
+import Home from './pages/Home'
+import Login from './pages/Login'
+import { Route, Routes } from 'react-router-dom'
+import { ApolloClient, InMemoryCache, ApolloProvider, HttpLink, from } from '@apollo/client'
+
+const link = from([new HttpLink({ uri: 'http://localhost:3000/graphql' })])
```

Comment:
Provide this link within an environment variable.
Sometimes the server URL might change, so we do not need to always change the code.